structs/ctl/message.go

```diff
@@ -7,8 +7,13 @@ import (
 )
 
 const (
-	CommandStart = "start"
-	CommandStop  = "stop"
+	CommandStart              = "start"
```

Comment:
do you think this list will get out of hand at some point or is it fine for now? I played around with making the requests arbitrary json and Request an interface so each command type could be different (for example some can take in args/flags but others can't) but then unmarshalling it gets kinda weird.

Reply:
Overall, I think its a good idea to have the structure of requests/responses be well defined, even if its a little annoying to deal with. It makes it a lot easier to figure out when things are going wrong, and if we ever need to create JSON/graphql/database schemas from the types we have it will be trivial